app/css/openmrs-addonmanager.css

```diff
@@ -126,10 +126,10 @@ a:hover {
 }
 
 .dropdown-menu-large>li>a:hover {
-    background-color: transparent;
-    margin-right: 0px;
-    border-bottom: 0px;
-    color: purple;
+    /* background-color: transparent;
```

Comment:
Could we not comment out this code and not have it in the codebase if it is not in use.